src/cobalt_model/pagination_config.rs

```diff
@@ -150,7 +153,7 @@ impl PaginationConfigBuilder {
         let permalink_suffix =
             permalink_suffix.unwrap_or_else(|| DEFAULT_PERMALINK_SUFFIX.to_owned());
         let order = order.unwrap_or(SortOrder::Desc);
-        let sort_by = sort_by.unwrap_or_else(|| vec![DEFAULT_SORT.to_owned()]);
+        let sort_by = sort_by.unwrap_or_else(|| DEFAULT_SORT.to_vec());
```

Comment:
I think `clone` would better convey intent but oh well

Reply:
I can change it, I love having your opinion on this kind of stuff, it teach me to better express the intent of my code :)